src/background/migration.js

```diff
@@ -0,0 +1,125 @@
+import {Component} from "./component.js";
+import {migrationData} from "./migrationData.js";
+import {Tier} from "./tier.js";
+
+const TIER_ENDPOINT_URLS = [
+ // [ URL, redirect to ]
+ [ "https://private-network-beta-landing-test.stage.mozaws.net/free-tier-enabled-install", "https://private-network-beta-landing-test.stage.mozaws.net/free-tier-enabled" ],
```

Comment:
Some of these are dev URLs and some are prod (which I _assume_ is on purpose), but... possibly worth noting is that the staging URL was meant to be internal and never public. /oops